pipelinerl/async_llm.py

```diff
@@ -43,6 +44,18 @@ def extract_images_from_messages(messages: list[dict]) -> list[Image.Image]:
     return images
 
 
+def _to_plain_obj(value):
+    """Recursively convert OmegaConf containers into standard Python types."""
+
+    if isinstance(value, (DictConfig, ListConfig)):
+        return OmegaConf.to_container(value, resolve=True)
```

Comment:
You can probably replace the whole `def _to_plain_obj` function with https://omegaconf.readthedocs.io/en/2.3_branch/usage.html#omegaconf-to-object